### PR TITLE
Add Hotkeys to chat

### PR DESCRIPTION
**Pull request by Wolvan**
Use Alt+[Down Arrow] to hide and Alt+[Up Arrow] to restore the chat

### CODE REVIEW

File: Extensions/messaging_tweaks.js [hunk 1]
```diff
@@ -267,6 +272,19 @@ XKit.extensions.messaging_tweaks = new Object({
 		if (!XKit.interface.is_tumblr_page()) {
 			return;
 		}
+		
+		if (XKit.extensions.messaging_tweaks.preferences.minimise_hotkey.value) {
+			$(document).on("keydown.minimise_header", function(e) {
+				if (e.which === 40) {
```
Comment (finagle29) [inline Extensions/messaging_tweaks.js]: this whole thing should be wrapped in an `if (e.altKey)` block

Comment (Wolvan) [inline Extensions/messaging_tweaks.js]: To only react when using alt? I thought I did that already, but seems not

**Comment by Wolvan** [comment]
Should be fixed, @finagle29

**Comment by finagle29** [comment]
works, LGTM @homu r+

**Pull request merged**